xbmc/Util.cpp

```diff
@@ -1596,7 +1596,7 @@ void CUtil::InitRandomSeed()
   srand(seed);
 }
 
-#ifdef TARGET_POSIX
+#if defined(TARGET_POSIX) && !defined(TARGET_DARWIN_TVOS)
```

Comment:
make one commit which changes `TARGET_DARWIN_IOS` -> `TARGET_DARWIN_EMBEDDED` and another one which adds needed preprocesseor  guards